### PR TITLE
[tools] Update android SDK & NDK install script

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates android SDK & NDK install script
- Remove NDK install path workaround for TFLite 1.13.1
- Update SDK initial commandline tool version to same version with dockerfile
- Update SDK commandline tool install process for easy version up
- Update runtime android build documentation

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by glistening** [comment]
@hseok-oh I succeed to run `./tools/cross/install_android_ndk.sh`. Where can I find the installed ndk?

File: tools/cross/install_android_ndk.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ do
     shift
 done
 
-__InstallDir=${__InstallDir:-"${__CrossDir}/ndk/${__NDKVersion}"}
+__InstallDir=${__InstallDir:-"${__CrossDir}/ndk"}
```
Comment (glistening) [inline tools/cross/install_android_ndk.sh:62]: Here, I checked where `ndk` is installed.

**Review by glistening — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**